code/modules/projectiles/guns/ballistic/revolver.dm

```diff
@@ -97,6 +97,11 @@
 	if(last_fire && last_fire + 15 SECONDS > world.time)
 		. = span_notice("[user] touches the end of [src] to \the [A], using the residual heat to ignite it in a puff of smoke. What a badass.")
 
+/obj/item/gun/ballistic/revolver/syndicate
+	name = "\improper .357 revolver"
+	desc = "A suspicious revolver. Uses .357 ammo."
```

Comment:
if its already set to that by parent no need to set it again

Reply:
I'm gonna change the name and description so they aren't the same as the normal revolver and to add some flavour